### ISSUE TITLE
A11y_Windows Terminal_About_Keyboard: All the links present in "About" pop up window are not accessible through keyboard even though focus is landing on it

### ISSUE THREAD

**Issue by SpandanaPaladi** [microsoft/terminal#5394]
#A11y_WindowsTerminal; #WindowsTerminal; #A11yMAS; #A11yTCS; #DesktopApp; #Win10;#C+E-H ; #Keyboard; #A11ySev2; #NonBenchMark; #MAS2.1.1

[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shcha1%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847087032&sdata=uWx6cTEfazn%2BoCV3cL1iYB4KN%2BuXQ/MTlmOVec%2BZgKc%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Version: 0.10.781.0

**Repro steps:**

1. Hit the URL ""aka.ms/terminal"" and install ""Windows terminal (Preview)"" application
2. Open ""Windows terminal (Preview)"" application
3. Tab to "Dropdown" control and press enter or use "Ctrl + Shift +  Space" 
4. Tab till "About" and press enter
5. Tab till the links present in "About" pop up window and press enter
6. Verify whether the links are accessible through keyboard or not

**Actual Result:​**
All the links present in "About" pop up window are not accessible through keyboard
​
**Expected Result:​**
All the links present in "About" pop up window should be accessible through keyboard
​
**User Impact:​**
Keyboard users will face difficulty if the links are not accessible through keyboard because they get block and access that information

**MAS Reference:​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Bug_Links not accessible through keyboard.pptx]
(https://github.com/microsoft/terminal/files/4492816/Bug_Links.not.accessible.through.keyboard.pptx)

**Comment by carlos-zamora** [comment]
I don't see any changes to the About dialog and it works just fine on 0.11.1072.0. The only thing I can think of is that we're getting the URI from the resources file. So if we're in a "locale" that isn't supported yet, I guess the link wouldn't launch?

**Comment by carlos-zamora** [comment]
Wait, but I guess then all the text would be missing too. So I guess not?

**Comment by carlos-zamora** [comment]
@SpandanaPaladi At 0:14 in your attached PowerPoint, the "Getting Started" link is already in focus. These links are expected to open a website on your web browser. Could you try reinstalling Windows Terminal and see if that helps?

**Comment by DHowett-MSFT** [comment]
I do not understand. The video shows that "Getting Started" does indeed have keyboard focus, and then later that "Documentation" receives keyboard focus. Is this not intended?

**Comment by SpandanaPaladi** [comment]
> @SpandanaPaladi At 0:14 in your attached PowerPoint, the "Getting Started" link is already in focus. These links are expected to open a website on your web browser. Could you try reinstalling Windows Terminal and see if that helps?

@carlos-zamora I have reinstalled the terminal and checked the issue once again but the issue is still reproducing. Version of the terminal is Version: 0.10.781.0

**Comment by Amulya-95** [comment]
Regressed the bug in latest environment. The issue resolved, now user is able to access links present in the 'About' dialog using keyboard. Hence closing this bug.
Please find the attachments for reference.

**Attachments:**
[[Fixed]Bug 5394_Able access links using keyboard.pptx](https://github.com/microsoft/terminal/files/4627813/Fixed.Bug.5394_Able.access.links.using.keyboard.pptx)
